packages/web-shared/components/Logo/Logo.js

```diff
@@ -7,11 +7,15 @@ import Styled from './Logo.styles';
 import { Box, systemPropTypes } from '../../ui-kit';
 
 function Logo({ fill, size, padding, theme, source, ...rest }) {
-  let themeData = '';
-  try {
-    themeData = JSON.parse(theme);
-  } catch (error) {
-    console.error('Error parsing JSON');
+  let themeData;
```

Comment:
This component was being passed an object, but was treating it like a string that needed parsed.